File: myheroesinkotlin/app/src/main/res/drawable/app/src/main/java/com/example/myheroes/network/Service.java
package com.example.myheroes.network;

import com.example.myheroes.model.HeroResults;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;
import retrofit2.http.Query;

public interface Service {

    @GET("/")
    Call<HeroResults> getHeroes(
            @Query( "hero" ) String hero,
            @Query( "rapidapi-key" ) String apiKey
    );
}

File: myheroesinkotlin/app/src/main/java/com/example/myheroesinkotlin/network/Service.kt
package com.example.myheroesinkotlin.network

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

import com.example.myheroesinkotlin.model.HeroResults

interface Service {

    @GET("/")
    fun getHeroes(
        @Query("hero") hero: String?,
        @Query("rapidapi-key") apiKey: String?
    ): Call<HeroResults?>?
}